{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -150,10 +158,6 @@ def get_targets_from_recipe():
 
 # COMMAND ----------
 
-client.get_model_version(model_name, model_version).description
```

Comment:
Did this cell previously serve to output the description to the user? And if so, why did we decide to not do that?

Reply:
I'm almost sure this is a command that I used for debugging and forgot to delete. It prints the current model version description, which the user doesn't care.

Reply:
Fantastic! Let's kill it 🙂